packages/server/src/campaign/campaign.controller.ts

```diff
@@ -24,4 +26,15 @@ export class CampaignController {
   public getCampaigns() {
     return this.campaignService.findAll();
   }
+
+  @Get(':id')
+  public getCampaign(@Param('id') id: string) {
+    return this.campaignService.findOne(id);
+  }
+
+  @Post('campaign-update/:id')
```

Comment:
Let's update this to:

```suggestion
  @Post(':id/updates')
```

So that the resulting URL would be `api/campaigns/123/updates` instead of `api/campaigns/campaign-update/123`